backend/app/Http/Controllers/UserController.php

```diff
@@ -68,4 +68,13 @@ public function userDetails()
 		$user = Auth::user();
 		return response()->json(['data' => $user]);
 	}
+
+	public function visitUser($user_id)
+	{
+		$user = User::find($user_id);
```

Comment:
Where is the other data such us user activities and followers?

Reply:
Naa sa katong activities api ug sa follow api nya sa integration na nako gidungan ang duha sir :bow: